scripts/package.lua

```diff
@@ -40,7 +40,17 @@
 -- Make sure I've got what I've need to be happy.
 --
 
-	local required = { "git", "make", "gcc", "premake5", "zip" }
+	local required = { "git" }
+
+	if not os.ishost("windows") then
+		table.insert(required, "make")
+		table.insert(required, "cc")
```

Comment:
The previous list had `gcc` but this is `cc`, is this correct?

Reply:
On both Linux and macOS, cc exists and is a symbolic link to gcc and clang respectively.